src/Core/Migration/V6_6/Migration1739355802FixVatHandling.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_6;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ *
+ * @codeCoverageIgnore
+ */
+#[Package('framework')]
+class Migration1739355802FixVatHandling extends MigrationStep
+{
+    /**
+     * @return array<string, string>
+     */
+    private const VAT_PATTERNS = [
+        'BE' => 'BE(\d{10}$|^0\d{9})',
+        'GR' => '(EL|GR)\d{9}',
+        'HR' => 'HR\d{11}',
+        'IE' => 'IE(\d[A-Z0-9]\d{5}[A-Za-z]$|^\d{7}[A-W][A-I])', // pre and post 2013 pattern
+        'LT' => 'LT(\d{9}|\d{12})',
+        'RO' => 'RO(?!0)\d{1,10}',
```

Comment:
what is this `(?!0)` in there? according to several websites, the correct one just has numbers (and possibly the `RO`)

Reply:
Never seen this. I would have expected to do a [1-9]\d{0,9}